src/pages/documentation/index.md

```diff
@@ -11,9 +11,9 @@ Welcome to the home of Adobe Experience Platform Mobile SDK documentation.
 
 <InlineAlert variant="success" slots="header, text"/>
 
-Good to go for iOS 17
+Good to go for iOS 17 and Android 14
 
-The latest versions of the Adobe Experience Platform SDK for Apple iOS (Swift and Objective-C) are tested for compatibility with the release version of Apple iOS 17. See [Current SDK Versions](./current-sdk-versions.md) to find our latest SDK versions, or [release notes](./release-notes/index.md) for information on changes and updates.
+The latest versions of the Adobe Experience Platform SDK for Google Android and Apple iOS (Swift and Objective-C) are tested for compatibility with the release versions of Google Android 14 and Apple iOS 17. See [Current SDK Versions](./current-sdk-versions.md) to find our latest SDK versions, or [release notes](./release-notes/index.md) for information on changes and updates.
```

Comment:
The latest versions of the Adobe Experience Platform SDK for Android and iOS (Swift and Objective-C) ...